tools/client-plugins/browser-scripts/frame-runner.ts

```diff
@@ -92,7 +92,13 @@ async function initTestFrame(e: InitTestFrameArg = { code: {} }) {
         // frameDocument ready:
         $(() => {
           try {
-            const test: unknown = eval(testString);
+            let test: unknown;
+            if (__file('script.js').indexOf(editableContents) > -1) {
```

Comment:
```suggestion
            if ((__file('script.js')?.indexOf(editableContents) ?? -1) > -1) {
```

This needs to be optional, and then nullish coalesced to make the `undefined` not pass the tests.